xAPI.md

```diff
@@ -507,13 +507,39 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
-## 8.0 Getting More Out of xAPI
 
-### 8.1 Extending xAPI
 
-### 8.2 Profiles
+## 8.0 Extending xAPI
+
+xAPI can be extended in a few ways.  The most notable and extensive is Statement Extensions, which allow great flexibility within Statements.  It is recommended that profiles or Communities or Practice agree on how to use  
+extensions for their particular use cases. Implementation details are covered in a [later section](#miscext)
+
+The About Resource is another place xAPI supports Extensions.  The LRS may find it useful to communicate features or behaviors beyond this specification to activity provider.  It is hoped that if more information is available about an LRS, that the About Resource contains it.
+
+Finally, the set of Resources implemented is not expected to be constrained by this document. Resources beyond 
+those listed in this specification can be implemented and co-exist with current Resources.
+
+<a name="COPs" />
+
+## 9.0 Profiles and Communities of Practice
+
+xAPI focuses on the structure of communication between a client and an LRS.  It spares many details of the content 
+to increase flexibility.  The intention in keeping the specification somewhat loose is that Communities of  
+Practice (CoP) can define content (verbs, activity types, contextual relationships, extensions, etc.) for their use cases.  It is very important that such communities exist and share best practices.
+
+A profile of xAPI is a set of rules and/or vocabularies applied to a valid xAPI implementation.  Often, a CoP leverages many best practices to create a profile.
+
+A profile will typically describe in greater detail some of the open-ended areas of the specification, such as 
+verbs, activity types, attachment usage, extensions, structure and domains for IRIs, and keys/possible values 
```

Comment:
This is repeating line 528. To avoid repetition, maybe this should just say something like "The CoP will document these identifiers in a profile." and then go straight on to the recommendation about the category context activity?